File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        numset=set(nums)
        longest=0
        for i in nums:
            if i-1 not in nums:
                current_length=0
                while i+current_length in numset:
                    current_length+=1
                longest=max(current_length,longest)
        return longest

File: Data Structures & Algorithms/duplicate-integer/submission-0.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        int a=0;
        for(int i : nums){
            a++;
            for(int j=a;j<nums.length; j++){
                if(i==nums[j]){
                    return true;
                }
            }
        }
        return false;
    }
}
